tests/util.go

```diff
@@ -12,6 +12,17 @@ import (
 	ktests "kubevirt.io/kubevirt/tests"
 )
 
+type Result struct {
```

Comment:
I am not sure why this struct and all fields need to be exported (public)?

Reply:
sorry, just a mistake, it was exported for some reason, but no need to anymore